teuthology/run_tasks.py

```diff
@@ -255,4 +255,3 @@ def notify_sleep_before_teardown(ctx, stack, sleep_time):
     smtp = smtplib.SMTP('localhost')
     smtp.sendmail(msg['From'], [msg['To']], msg.as_string())
     smtp.quit()
-
```

Comment:
@shyukri this change is not related, could you please undo this?

Reply:
Done. VS Code was misbehaving.